checklist/src/main/scala/checklist/Checked.scala

```diff
@@ -0,0 +1,138 @@
+package checklist
+
+import cats.implicits._
+import cats.{Semigroup, Monad, Monoid}
+import cats.data.{NonEmptyList, Ior, Validated}
+
+sealed trait Checked[E, W, R] {
+  def widen[EE >: E, WW >: W, RR >: R]: Checked[EE, WW, RR] = this.asInstanceOf[Checked[EE, WW, RR]]
+  def widenErrors[EE >: E]: Checked[EE, W, R] = this.asInstanceOf[Checked[EE, W, R]]
+  def widenWarnings[WW >: W]: Checked[E, WW, R] = this.asInstanceOf[Checked[E, WW, R]]
+  def widenResult[RR >: R]: Checked[E, W, RR] = this.asInstanceOf[Checked[E, W, RR]]
```

Comment:
I don't like how I'm doing any of these widenings.  Should change to liskov

Reply:
With covariance these would just be `= this`, right?

Reply:
With covariance, all of these wouldn't even need to exist at all... The haskell within me is just telling me to avoid variance. 😅 I think I'll just have to get over it.